media-tv/plex-media-server/plex-media-server-1.13.9.5456.ebuild

```diff
@@ -0,0 +1,135 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit user systemd unpacker python-single-r1
+
+DESCRIPTION="A free media library that is intended for use with a plex client."
+HOMEPAGE="http://www.plex.tv/"
+
+COMMIT="ecd600442"
+
+_APPNAME="plexmediaserver"
+_USERNAME="plex"
+_SHORTNAME="${_USERNAME}"
+_FULL_VERSION="${PV}-${COMMIT}"
+
+URI="https://downloads.plex.tv/plex-media-server"
```

Comment:
Strictly not required, but you can move stuff from `SRC_URI` to this line (and this variable might need renaming, but wait for a dev to confirm it).